lib/compat/wordpress-6.2/class-gutenberg-rest-global-styles-controller-6-2.php

```diff
@@ -89,6 +89,28 @@ public function prepare_item_for_response( $post, $request ) { // phpcs:ignore V
 		return $response;
 	}
 
+	/**
+	 * Get the link relations available for the post and current user.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @return array List of link relations.
+	 */
+	protected function get_available_actions() {
+		$rels = array();
+
+		$post_type = get_post_type_object( $this->post_type );
+		if ( current_user_can( $post_type->cap->publish_posts ) ) {
+			$rels[] = 'https://api.w.org/action-publish';
+		}
+
+		if ( current_user_can( 'edit_css' ) ) {
+			$rels[] = 'https://api.w.org/action-edit-css';
+		}
```

Comment:
I couldn't find documentation on how these URLs are generated/created. So I need to double-check if `https://api.w.org/action-edit-css` is the correct format.